browser/browser_context_keyed_service_factories.cc

```diff
@@ -188,6 +189,7 @@ void EnsureBrowserContextKeyedServiceFactoriesBuilt() {
 
   ai_chat::AIChatServiceFactory::GetInstance();
   ai_chat::ModelServiceFactory::GetInstance();
+  ai_chat::TabTrackerServiceFactory::GetInstance();
```

Comment:
shouldn't this (and AIChatServiceFactory for that matter) be behind `ai_chat::features::IsAIChatEnabled`?

Reply:
Good call!